app/views/page_not_found_template.scala.html

```diff
@@ -14,28 +14,31 @@
  * limitations under the License.
  *@
 
-@import uk.gov.hmrc.play.views.html.layouts.Article
 @import config.ApplicationConfig
+@import views.html.govuk_wrapper
 
-@this(article: Article)
+@this(
```

Comment:
I can't find this page in use anywhere, is it still needed?

Reply:
It should be used, but as we discussed this isn't a regressive change. Will raise a ticket.